pkg/persistence/disk_persistence.go

```diff
@@ -149,6 +149,13 @@ func (ds *diskPersistence) Archive(directory string) error {
 	return moveAll(from, to)
 }
 
+func (ds *diskPersistence) Delete(dirName string, fileName string) error {
+	dirPath := ds.getStorageCurrentDirPath()
+	filePath := filepath.Join(dirPath, dirName, fileName)
```

Comment:
Can we take the opportunity and update all `fmt.Sprintf("%s/%s/%s",...)` occurrences in this file?

Reply:
41ca4fdc902a4c0b4c13d0007d811859bf9c9ba0